src/en/news/blog/2013/a-gentle-introduction-to-the-erasure-coding/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "A gentle introduction to the erasure coding"
+date: "2013-09-09"
+author: "shan"
+tags: 
+---
+
+![](images/erasure-coding-intro.jpg "A gentle introduction to the erasure coding")
+
+The erasure coding is currently a very hot topic for distributed storage systems. It has been part of the Ceph roadmap for almost a year and Swift guys recently brought the discussion to the table. Both of them have planned to implement the erase code functionality so I thought it might be interesting to give a high level overview about the erasure coding principles. Before we start, I’d like to point out that **I don’t take any credit for this article**. I just read the wonderful white paper [“Erasure Coding vs. Replication: A Quantitative Comparison”](http://www.cs.rice.edu/Conferences/IPTPS02/170.pdf) written by Hakim Weatherspoon and John D. Kubiatowicz from Computer Science Division University of California, Berkeley. Many thanks to them. While ready the paper, I found their introduction to the erasure code easy to understand for a novice like me :).
+
+  
+
+# I. Introduction
```

Comment:
I appreciate it's unrealistic to go through all the old articles and update the headings so they are semantic but this article has multple `h1` tags.

Reply:
Updated.